### PR TITLE
Updating DG

### PR DESCRIPTION
**Pull request by 9fc70c892**

### CODE REVIEW

**Comment by 9fc70c892** [comment]
- [ ] Checked images/ComponentManagers.png
- [ ] Updated images/ComponentManagers.png

- [ ] Checked images/UiClassDiagram.png
- [ ] Updated images/UiClassDiagram.png

- [ ] Checked images/LogicClassDiagram.png
- [ ] Updated images/LogicClassDiagram.png

- [ ] Checked images/ParserClasses.png
- [ ] Updated images/ParserClasses.png

- [ ] Checked images/ModelClassDiagram.png
- [ ] Updated images/ModelClassDiagram.png

- [ ] Checked images/StorageClassDiagram.png
- [ ] Updated images/StorageClassDiagram.png

File: docs/diagrams/ModelClassDiagram.puml [hunk 1]
```diff
@@ -5,46 +5,37 @@ skinparam arrowColor MODEL_COLOR
...
-Class Email
-Class Name
-Class Phone
-Class Tag
-
+Class UniqueXList
+Class X
+Class PartMap
+'Class Part
 }
```
Comment (kimberlybp) [inline docs/diagrams/ModelClassDiagram.puml:19]: Think you missed out `ServiceList which is also something `Shop` also has an association with, ServiceList is implemented similarly to `UniqueXList` but without the unique constraint within the list

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -154,6 +157,37 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+
+### Current Implementation
+
+The view function is facilitated by `ViewXCommand` (`X` is a placeholder for the specific command name e.g., `ViewCustomerCommand`).
+
+Here X can be appointment/customer/service/technician/vehicle.
+
+The Sequence Diagram below illustrates the interactions within the Logic component for the execute("viewcustomer 1") API call.
+
+<img src="images/ViewSequenceDiagram.png" width="600"/>
```
Comment (kimberlybp) [inline docs/DeveloperGuide.md:170]: Do remember to check if the size of the diagram is big enough to be displayed clearly, think 600 may be too small, couldn't really see the labels when I looked at the md file from github

File: docs/diagrams/ModelClassDiagram.puml (hunk 1 above)
Comment (9fc70c892) [reply docs/diagrams/ModelClassDiagram.puml:19]: like dis?
image

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (9fc70c892) [reply docs/DeveloperGuide.md:170]: ops

File: docs/diagrams/ModelClassDiagram.puml (hunk 1 above)
Comment (kimberlybp) [reply docs/diagrams/ModelClassDiagram.puml:21]: Y can just be Service(?)

**Pull request merged**